FWCore/Integration/test/SwitchProducer_t.cpp

```diff
@@ -234,11 +234,16 @@ TEST_CASE("Configuration with different branches", s_tag) {
     edm::test::TestProcessor::Config config1{baseConfig1};
     REQUIRE_THROWS_WITH(
         edm::test::TestProcessor(config1),
-        Catch::Contains("that does not produce a product") && Catch::Contains("that is produced by the chosen case"));
+        Catch::Contains("that does not produce a product") && Catch::Contains("that is produced by the chosen case") &&
+            Catch::Contains("Products for case s@test1") && Catch::Contains("Products for case s@test2") &&
+            Catch::Contains("edmtestIntProduct \n") && Catch::Contains("edmtestIntProduct foo"));
 
     edm::test::TestProcessor::Config config2{baseConfig2};
-    REQUIRE_THROWS(edm::test::TestProcessor(config1),
-                   Catch::Contains("with a product") && Catch::Contains("that is not produced by the chosen case"));
+    REQUIRE_THROWS_WITH(
+        edm::test::TestProcessor(config2),
```

Comment:
This one (and similar change below) actually fix a bug in the test as well. This test should have used `config2` already before, but was not caught with the `Catch::Contains` because the test mistakenly used `REQUIRE_THROWS` instead of `REQUIRE_THROWS_WITH`.